### PR TITLE
Remove ORM

### PR DESCRIPTION
**Pull request by htunnicliff**
I removed TypeORM, added SQLite, and switched to Hono, which is a really simple HTTP framework that would be easily usable by anyone who has worked with Express, Koa, Fastify, or other similar ones.

The tasks mentioned in the instructions include testing, so if we want to support that, we should probably add a Jest configuration.

### CODE REVIEW

File: PLEASE_READ_FIRST.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+The purpose of this exercise is to evaluate your backend skills in Node.js, SQL, and testing. Please note that you may make assumptions, simplifications, or other changes to the problems, but please state them clearly in your write-up when you submit this assignment. Please feel free to use libraries as appropriate, as there is no need to reinvent the wheel.
+
+Before starting, please review the instructions carefully.
+
+**Good luck 🙃**
+
+# Acceptance criteria
+
+Your task is to build a REST API usingSQLite to manage a property list.
```
Comment (GentryRiggen) [inline PLEASE_READ_FIRST.md]:
```suggestion
Your task is to build a REST API using SQLite to manage a property list.
```

File: PLEASE_READ_FIRST.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+The API should enable users perform CRUD actions (create, read, update, delete) on property data from the database.
```
Comment (GentryRiggen) [inline PLEASE_READ_FIRST.md]:
```suggestion
The API should enable users to perform CRUD actions (create, read, update, delete) on property data from the database.
```

File: PLEASE_READ_FIRST.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Requests should be validated where applicable.
+
+Invalid requests and other errors should be handled appropriately along with the appropriate HTTP status codes. You may use your discretion regarding error responses themselves.
+
+Where applicable, endpoints returning multiple records should be filterable and support pagination. You may choose any pagination strategy you are familiar with.
```
Comment (GentryRiggen) [inline PLEASE_READ_FIRST.md:23]: I'm still thinking we don't require filtering and that is asked to be added in the pair programming interview. Thoughts @a88zach @xdc0 @dbaq

File: src/index.ts [hunk 4]
```diff
@@ -1,24 +1,51 @@
...
-import { router } from "./routes/propertyRoutes"
-import { Property } from './entities/Property'
+// =============== //
+// Initialize app  //
+// =============== //
 
-const app = express();
-const PORT = process.env.PORT || 3000;
+// Docs for Hono: https://hono.dev
+const app = new Hono();
```
Comment (GentryRiggen) [inline src/index.ts:10]: I feel like we should just use express

File: src/index.ts [hunk 5]
```diff
@@ -1,24 +1,51 @@
...
+  return c.text("Create property");
+});
+
+app.put("/properties", (c) => {
+  return c.text("Update property");
+});
+
+app.delete("/properties", (c) => {
+  return c.text("Delete property");
+});
```
Comment (GentryRiggen) [inline src/index.ts]: Throw in these instead?

File: src/index.ts [hunk 6]
```diff
@@ -1,24 +1,51 @@
...
+
+// ========== //
+// Start app  //
+// ========== //
+
+serve(app);
+
+console.log("Server started at http://localhost:3000");
```
Comment (GentryRiggen) [inline src/index.ts]: Definitely should have a 🚀

File: src/types.ts [hunk 7]
```diff
@@ -0,0 +1,8 @@
+export type Property = {
```
Comment (GentryRiggen) [inline src/types.ts]: I don't think we should define any types so we can see what they do

File: src/index.ts (hunk 5 above)
Comment (htunnicliff) [reply src/index.ts]: Not sure what you mean – was there a code block?

Comment (GentryRiggen) [reply src/index.ts]:
```suggestion
app.get("/properties", (c) => {
  throw new Error('NOT IMPLEMENTED');
});

app.get("/properties/:id", (c) => {
  throw new Error('NOT IMPLEMENTED');
});

app.post("/properties", (c) => {
  throw new Error('NOT IMPLEMENTED');
});

app.put("/properties", (c) => {
  throw new Error('NOT IMPLEMENTED');
});

app.delete("/properties", (c) => {
  throw new Error('NOT IMPLEMENTED');
});
```

Comment (htunnicliff) [reply src/index.ts]: Added

File: src/index.ts (hunk 4 above)
Comment (htunnicliff) [reply src/index.ts:10]: We definitely could, but Hono has some nice out-of-the-box stuff that both Fastify and Express require plugins for.

**Pull request closed without merging**